internal/kafka/constants/kafka.go

```diff
@@ -130,3 +130,13 @@ func GetSuspendedStatuses() []string {
 func GetDeletingStatuses() []string {
 	return []string{KafkaRequestStatusDeprovision.String(), KafkaRequestStatusDeleting.String()}
 }
+
+type BillingModel string
```

Comment:
Can we call it `KafkaBillingModel`?
We currently have several billing model levels (kafka billing model, ams billing model, quota management lis billing model, ...) and this would help us differentiate them

Reply:
> I am assuming this refers to a Kafka billing model and not to an AMS billing model. Could you tell me which one is the case?

The assumption is correct. This is a kafka billing model not ams billing model.

> Assuming it is the former, can we call it KafkaBillingModel?

There is already a `KafkaBillingModel`, maybe a rename to `KafkaBillingModelID` might be best suitable, what do you think?

> We currently have several billing model levels (kafka billing model, ams billing model, quota management lis billing model, ...) and this would help us differentiate them

I agree.

Reply:
If there is already a `KafkaBillingModel` why are we not using that?

Reply:
It's two different concepts. This code wants the ID of the billing model.

Reply:
The ID is taken from the configuration